content/geoip/docs/databases/anonymous-ip.mdx

```diff
@@ -122,6 +122,10 @@ These are named `GeoIP2-Anonymous-IP-Blocks-IPv4.csv` and
 
 <ConversionUtility />
 
+## Database Changes
+
+<DatabaseChanges product={"Anonymous IP"} />
```

Comment:
I don't think wrapping `"Anonymous IP"` in `{}` is necessary here. What happens if the braces are removed?